cairo/src/font/font_face.rs

```diff
@@ -64,14 +69,16 @@ impl FontFace {
     /// `FontFace` is dropped.
     #[cfg(any(feature = "freetype", feature = "dox"))]
     #[doc(alias = "cairo_ft_font_face_create_for_ft_face")]
-    pub unsafe fn create_from_ft(
```

Comment:
If you're turning this into a safe method, you should remove the `# Safety` section from the docs